alicloud/resource_alicloud_drds_instance_test.go

```diff
@@ -0,0 +1,121 @@
+package alicloud
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/drds"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/terraform-providers/terraform-provider-alicloud/alicloud/connectivity"
+)
+
+func TestAccAlicloudDRDSInstance_Basic(t *testing.T) {
```

Comment:
Please add a sweeper test for the resource.

Reply:
fix